src/js/routes/factories/network.js

```diff
@@ -111,7 +111,8 @@ let RouteFactory = {
                 parentCrumb: 'virtual-networks-tab-detail-tasks-detail',
                 getCrumbs: function () { return []; }
               }
-            }
+            },
+            title: 'Details'
```

Comment:
I'm not opposing this, however it really feels like we're overloading our route configuration objects.